sycl/include/CL/sycl/builtins.hpp

```diff
@@ -1554,6 +1554,15 @@ extern SYCL_EXTERNAL void __assert_fail(const char *expr, const char *file,
 }
 #elif defined(_WIN32)
 extern "C" {
+extern SYCL_EXTERNAL double _Cosh(double x, double y);
```

Comment:
Why do we need to declare these here?  I thought they are declared in Mircosoft header files that provide std::complex support.

Reply:
Hi, @vzakhari .
Yes, as I supposed, MSVC implementation should declare those functions before using in std::complex implementation. Unfortunately, they didn't do as we expected. This problem was exposed when compiler added check for such behavior. And according to current checking rules, we need to add "SYCL_EXTERNAL" before such functions.